config/image/oss/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ images:
   newTag: '3.8' # renovate: datasource=docker versioning=docker depName=kong@regenerate packageName=kong
 - name: kic-placeholder
   newName: kong/kubernetes-ingress-controller
-  newTag: '3.3' # renovate: datasource=docker versioning=docker depName=kong/kubernetes-ingress-controller@regenerate packageName=kong/kubernetes-ingress-controller
+  newTag: '3.4' # renovate: datasource=docker versioning=docker depName=kong/kubernetes-ingress-controller@regenerate packageName=kong/kubernetes-ingress-controller
```

Comment:
well, it is part of the release process, and it should break only when installed from main, right?

Reply:
I see. I'd suggest that we consider changing this. I can see that these lines do contain the renovate comments so I'd expect these to be updated automatically by renovate's pr.

On the release branch this would be done manually after the image is pushed to a registry (as we don't have renovate configured on release branches and we shouldn't).

We can discuss this after 3.4 is done so that this change is incorporated starting from 3.5. WDYT?

Reply:
Sounds good to me. I'll create an issue for that.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/issues/6867